core/Macros.carp

```diff
@@ -294,3 +294,13 @@
 (defmacro save-docs [:rest modules]
   ;; A trick to be able to send unquoted symbols to 'save-docs'
   (list 'save-docs-internal (list 'quote modules)))
+
+(defndynamic project-config [bindings] 
+  (if (< (length bindings) 2)
```

Comment:
I’d like to add a check for `(= length 1)` here, just to make sure the user didn’t pass an odd number of key/value pairs.